app-frontend/src/app/components/scenes/sceneImportModal/sceneImportModal.controller.js

```diff
@@ -2,13 +2,17 @@
 /* global document */
 /* global window */
 
-const availableImportTypes = ['local', 'S3'];
+import planetLogo from '../../../../assets/images/planet-logo-light.png';
```

Comment:
:man_facepalming: I couldn't find this logo when doing the token frontend + api. can you change the one there to this so we don't need the dark background?